build.xml

```diff
@@ -10,19 +11,30 @@
 
 	<property name="version" value="0.0.0" />
 
-	<property name="build.dir" value="./bin" />
-	<property name="build.dir.compile" value="${build.dir}/classes" />
+	<property name="build.dir" value="./bin/main" />
+	<property name="build.dir.compile" value="${build.dir}/" />
 	<property name="build.dir.resources" value="${build.dir.compile}" />
 
-	<property name="test.build.dir" value="./bin"/>
-	<property name="test.build.dir.report" value="${test.build.dir}/report" />
-	<property name="test.build.dir.compile" value="${test.build.dir}/classes" />
+	<!-- Tests -->
+	<property name="test.groups" value="" />
+	<property name="test.excludegroups" value="long-runtime" />
```

Comment:
Some of the tests actually run the entire repository set. For example, the Zoo repository is about 10 minutes to do just that, so I let the build server run those. It can be overridden by just doing `-Dtest.excludegroups=""` on the command-line though. Travis runs *all* tests when it builds. 

They are always tweakable locally, just the default behaviour is to ignore them.